vscode-wpilib/eslint.config.js

```diff
@@ -0,0 +1,5 @@
+// eslint.config.js
+module.exports =
+  {
+      ignores: [".media/"]
```

Comment:
Shouldn't have the `.` in front of media

Reply:
I am just going off this: https://eslint.org/docs/latest/use/configure/ignore#ignoring-directories

Reply:
looks like you were right per usual.  Thanks for the help!

Reply:
I think that example was for a hidden directory (in linux hidden directories are prefixed with a period)